fem/libceed_mass.cu

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) 2010, Lawrence Livermore National Security, LLC. Produced at
```

Comment:
I suggest we reorganize these files as follows:
- make a subdirectory `fem/libceed`
- `libceed.*` -> `libceed/ceed.*`
- `libceed_mass.*` -> `libceed/mass.*`
- `libceed_diffusion.*` -> `libceed/diffusion.*`